DRCollectionViewTableLayout/DRCollectionViewTableLayoutManager.m

```diff
@@ -189,4 +189,23 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
     }
 }
 
+
+- (void)collectionView:(UICollectionView *)collectionView didDeselectItemAtIndexPath:(NSIndexPath *)indexPath {
+    NSUInteger row = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout rowNumberForIndexPath:indexPath];
+    NSUInteger column = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout columnNumberForIndexPath:indexPath];
+    
+    [self.delegate collectionViewTableLayoutManager:self
```

Comment:
Because `collectionViewTableLayoutManager:collectionView:didDeselectCellAtRow:column:indexPath:` method is defined as optional, delegate don't have to implement it. Before calling this method on delegate, check if it's implemented using `respondsToSelector:` method.